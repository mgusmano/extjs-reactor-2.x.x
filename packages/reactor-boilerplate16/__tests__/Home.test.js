import React from 'react';
import { create } from 'react-test-renderer';
import Home from '../src/Home/Home';

describe('Home', () => {
    it('should render without crashing', () => {
        const result = create(<Home/>);
        expect(result).toMatchSnapshot();
    })
});