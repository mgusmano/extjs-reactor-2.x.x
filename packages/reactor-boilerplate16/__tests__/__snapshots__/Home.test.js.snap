// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render without crashing 1`] = `
<Grid
  store={
    Object {
      "$className": "Ext.data.Store",
    }
  }
>
  <Toolbar
    docked="top"
  >
    <SearchField
      onChange={[Function]}
      placeholder="Search..."
      responsiveConfig={
        Object {
          "width > 576": Object {
            "flex": 1,
          },
          "width > 768": Object {
            "flex": undefined,
          },
        }
      }
      ui="faded"
    />
  </Toolbar>
  <Column
    dataIndex="name"
    flex={2}
    resizable={true}
    text="Name"
  />
  <Column
    dataIndex="email"
    flex={3}
    resizable={true}
    responsiveConfig={
      Object {
        "width > 576": Object {
          "hidden": true,
        },
        "width > 768": Object {
          "hidden": false,
        },
      }
    }
    text="Email"
  />
  <Column
    dataIndex="phone"
    flex={2}
    resizable={true}
    text="Phone"
  />
</Grid>
`;
